gui/web/src/components/molecules/Welcome/Welcome.js

```diff
@@ -48,7 +48,7 @@ class Welcome extends Component {
 
     const kelpLogo = (
       <div className={styles.image}>
-        <img className={styles.symbol} src={symbol} alt="Kelp Symbol"/>
+        <img className={styles.symbol} src={symbol} alt="Kelp Symbol" />
```

Comment:
can you configure your IDE to ignore whitespace changes for `.js` files?

The `.js` files are very inconsistent and PRs will be very noisy for both of us without this. It's fine for this commit.

Reply:
Got it. Will enable for subsequent PRs.